### ISSUE TITLE
Allow passing custom projectRoot option

### ISSUE THREAD

**Issue by TimonLukas** [KusStar/vite-bundle-visualizer#20]
Hi! Thanks for making & publishing this project, works very well! Just running into one issue - could you add an option for passing a custom `projectRoot` (single one would work for me)?

I'm using it in a monorepo for a sub-sub-package, so the `process.cwd` is deeper than the root of most modules. This means that most imported packages have the absolute path, which is a bit distracting when publishing the output in CI. The project structure is something like this:

```
/package.json                              <--- empty root
/node_modules/...                          <--- shared packages are here (e.g. typescript)

/packages/a/package.json                   <--- indirect dependency through b
/packages/a/node_modules/...               <--- most modules are here

/packages/b/package.json                   <--- implementation
/packages/b/node_modules/...               <--- some modules are here

/packages/b/examples/example1/             <--- project.cwd for vite-bundle-visualizer
/packages/b/examples/example1/package.json <--- Vite project
```

Calling the command from the root directory and passing in `--config` / `--entry` doesn't work either, as the visualizer can't import Vite.

If you'd like me to create a PR, I can gladly do so! :)

**Comment by KusStar** [comment]
Currently it's import `vite` from cwd and goes down in trees directly, maybe sometimes it cannot import indirectly vite from another tree. It's great if you would make a PR.

You might want to revise the line that mentions Vite being imported from the current working directory (cwd). I’m uncertain whether importing Vite from a different root directory will allow the build correctly, given that the entry file is located in a different root.

https://github.com/KusStar/vite-bundle-visualizer/blob/713be97f57b891efea9367dc048eceda145130f8/index.js#L10

**Comment by TimonLukas** [comment]
I looked into this issue further today and tried to build a reproduction case (as a test fixture). The simplest case didn't work, so I looked further into my specific problem case.

Somehow the issue seems to be that the Vite-generated source maps contain transformed source code, but not the finished version - some in-between version with dynamic imports of absolute module paths for workspace dependencies. This didn't occur in the bundle itself. What worked to "fix" the issue was adding:

```typescript
build: {
  sourcemap: true,
  rollupOptions: {
    output: {
      sourcemapExcludeSources: true,
    }
  }
}
```

to my Vite config - this removes that half-transformed source from the source map, which makes the bundle visualizer output the correct (short) path.

If you want to see the example yourself, I can give you access to the private repo where this happened. Other than that, this issue is resolved - might be worth to add the resolution to the README, in case someone else has the same issue.

Thanks again for this package!
